src/js/view/controls/components/menu/category-button.js

```diff
@@ -0,0 +1,39 @@
+import button from 'view/controls/components/button';
+import { cloneIcon } from 'view/controls/icons';
+import { SimpleTooltip } from 'view/controls/components/simple-tooltip';
+import { createElement } from 'utils/dom';
+
+const categoryButton = (menu, localizedName) => {
+    const name = menu.name;
+    const icons = {
+        captions: 'cc-off',
+        audioTracks: 'audio-tracks',
+        quality: 'quality-100',
+        playbackRates: 'playback-rate',
+    };
+
+    // Allows override from Commercial Sharing menu
+    let icon = icons[name];
+    if (!icon) {
+        icon = icons.quality;
+    }
+    const menuCategoryButton = button(
+        `jw-settings-${name}`, 
+        (event) => {
+            menu.open(event);
+        }, 
+        name, 
+        [(menu.icon && createElement(menu.icon)) || cloneIcon(icon)]
+    );
+
+    const buttonElement = menuCategoryButton.element();
+    buttonElement.setAttribute('role', 'menuitemradio');
+    buttonElement.setAttribute('aria-checked', 'false');
+    buttonElement.setAttribute('aria-label', localizedName);
+    if (!('ontouchstart' in window)) {
```

Comment:
It's a tricky one. I ported that from the old category button, but it's flawed. From MDN:

_This event model – which has since been submitted for W3C standardisation – unifies “pointer” devices (mouse, stylus, touch, etc) under a single new class of events. As this model does not, by design, include any separate ‘touch’, the feature detection for ontouchstart will naturally not work. The suggested method of detecting if a browser using Pointer Events is running on a touch-enabled device instead involves checking for the existence and return value of navigator.maxTouchPoints_

We could do something like the iPadOS detection and go off of navigator.maxTouchPoints. Let's discuss later today, I do like the idea of adding touch feature detection but I want to make sure I don't leave out a potential touch supported case.

Reply:
I made the comment because I want to avoid feature detection that hits browser globals like `window` in places like this. Whether or not there is pointer support is a different issue, but is related to whether or not we want to show tooltips.